src/libs/protocols/hack/update-generated-proto.sh

```diff
@@ -47,7 +47,7 @@ show_usage() {
 }
 
 generate_go_sources() {
-    local cmd="protoc -I$GOPATH/src:$GOPATH/src/github.com/kata-containers/kata-containers/src/libs/protocols/protos \
+    local cmd="protoc -I$GOPATH/src:$(realpath ../)/libs/protocols/protos \
```

Comment:
hi @beraldoleal !

This change still assume you are running from the `src/<component>`.  I suggest to use the path relative to this script's directory. Something like:
```
script_dir="$(dirname "$(readlink -f "$0")")"
local cmd="protoc -I$GOPATH/src:${script_dir}/../protos \
```

Reply:
I didn't want to change the current logic here. There is an assumption the script must be executed from `src/agent`. There are other places talking about it.

Here I just would like to make sure it is fetching the source from workdir instead of GOPATH.

I agree with your proposal, but I hadn't time to investigate why this is a requirement.